### DISCUSSION TITLE
packer-examples-for-fusion?

### DISCUSSION THREAD

**Discussion by vvlasenko** [vmware/packer-examples-for-vsphere#966] [Help] [answered]
Hey guys, this colletion of templates is great, but it  works only with vsphere. Is there something like this for Vmware Fusion available? Wold be very helpful for my current project. Thanks in advance. vad

**Comment by tenthirtyam** [comment, marked as answer]
At this time we don't have a similar project either and there are no plans to extend the scope of this project to cover the desktop hypervisors.
